views/pages/addTechnology.ejs

```diff
@@ -89,7 +93,7 @@
 <script>
 
     $("form").submit(function (event) {
-
+        $(":submit").attr("disabled", true);
```

Comment:
It should be inactive the moment the person clicks it - its always good practise to have that as the first action. It won't matter in this case whether we are disabling it as the first command, or we are disabling it after validation checks, but if the validation checks and further processing were performed asynchronously we could have pressed it twice before it changed itself to be disabled.